src/cf-layout.less

```diff
@@ -1404,6 +1404,289 @@
+              background-size: 150px;
+              width: 195px;
+            }
+          }
+        </style>
+        <section class="content_hero hero hero__wide-image">
+            <div class="hero_wrapper wrapper wrapper__match-content">
+                <div class="hero_text">
+                    <h1 class="hero_heading">Hero title</h1>
+                    <p class="hero_subhead">
+                        Hero text goes here
+                    </p>
+                    <p class="hero_cta">
+                        <a>Call to action</a>
+                    </p>
+                </div>
+            </div>
+        </section>    
+      notes:
+        - "If the illustration is wide enough to take up the whole space allotted
+           for it, add the wide-image class to position the illustration at the top
+           right, rather than center of the image area."
+      codenotes:
+        - .hero__wide-image    
+    - name: .hero__small-content modifier
+      markup: |
+        <style>
+          .hero {
+            background-color: #ccdfdf !important;
+          }
+          .hero_wrapper:after {
+            background-image: url("http://placekitten.com/g/400/195");
+            background-size: 350px;
+          }
+          .customized-hero .hero_wrapper {
+              padding-bottom: 225px;
+          }
+          @media only all and (min-width: 37.5em) {
+           .hero .hero_wrapper {
+              padding-right: 200px; 
+              padding-bottom: 30px;
+            }
+            .hero .hero_wrapper:after {           
+              background-size: 150px;
+              width: 195px;
+            }
+          }
+        </style>
+        <section class="content_hero hero hero__small-content">
+            <div class="hero_wrapper wrapper wrapper__match-content">
+                <div class="hero_text">
+                    <h1 class="hero_heading">Hero title</h1>
+                    <p class="hero_subhead">
+                        Hero text goes here
+                    </p>
+                </div>
+            </div>
+        </section>    
+      notes:
+        - "If the hero text is short or doesn't include a call to action, adding
+           the small-content class will increase the top padding value so the
+           text is more vertically aligned with the image."
+      codenotes:
+        - .hero__small-content
+  tags:
+    - cf-layout
+*/
+
+@hero-desktop-height: 300px;
+@hero-image-height: 195px;
+@desktop-image-container-width: 475px; //width of 5 of 12 columns at 1140px
+@tablet-image-container-width: 333px; //width of 5 of 12 columns at 799px
+
+.hero {
+  background-color: @gray-10;
+  
+  &_wrapper {
+    position: relative;
+    padding-top: unit(@grid_gutter-width / @base-font-size-px, em);
+    // bottom padding = image height at mobile + padding for top/bottom of image
+    padding-bottom: unit((@hero-image-height + @grid_gutter-width * 2) / @base-font-size-px, em); 
```

Comment:
Ignore, after playing with this locally I realized this is for smaller screens where the images is stacked below.